### DISCUSSION TITLE
Remove legacy code in updater

### DISCUSSION THREAD

**Discussion by HLeithner** [joomla/joomla-cms#47040] [3. Approved Features]
Joomla updater has code for old CMS Version, was useful for Versions < 4.0

https://github.com/joomla/joomla-cms/blob/dfb572de83d16a25e7eee673b3875e18750e688b/administrator/components/com_joomlaupdate/src/Model/UpdateModel.php#L175-L179

I would expect that this code part could be removed.

# Approval Information
Approved: 25.02.2026
Minimum Target Version: 7.0

**Comment by Hackwar** [comment]
Marking this as feature, since it isn't technically a bug and would have to be done in a minor or major version.

**Comment by rdeutz** [comment]
The Joomla Maintainer Teams decided on the 25.02.2025 meeting to accept the feature.
